### DISCUSSION TITLE
bytes is not part of the body mixin

### DISCUSSION THREAD

**Discussion by woss** [nodejs/undici#3402] [Q&A] [answered]
It seems that the `body.bytes()` doesn't exist in types.
<img width="664" alt="image" src="https://github.com/user-attachments/assets/9e778339-c826-432c-b707-ab8919962c13">

but `.arrayBuffer()` does
<img width="683" alt="image" src="https://github.com/user-attachments/assets/051d1fe5-ab1f-467f-ab7f-8e379a4ab401">

**Comment by tsctx** [comment, marked as answer]
I implemented it in #3391, but it has not been released yet.

**Reply by woss** [reply]
interesting that the README.md is released with the info that says the bytes() are available
